internal/env/scm.go

```diff
@@ -46,6 +50,24 @@ func ClientSecret() (string, error) {
 	return clientSecret, nil
 }
 
+// AppID returns the appliction ID for the current SCM provider.
```

Comment:
Does it make sense to provide a default path and AppID? Similar to `CertFile` and `KeyFile`.

Reply:
I can add a default file path, but then I must know what the default file name will be (the .pem file must be generated on the app's web page and then saved, it must be done manually). 

`AppID` is an app-specific variable just like `clientID` and `clientSecret` (OAuth apps had only two (ID, secret), GitHub apps have three (app ID, client ID, client secret). Providing a default `appID` would expose the ID of the GitHub App created for UIS.

Reply:
Seems like providing a default file path makes sense. The user would be expected to place the `.pem` file at the default location, unless another path is provided via the env var.

Another thing I've been thinking about in relation to this. It seems reasonable that these clientID, clientSecret, appID and so on could be bundled into a struct that can be passed around instead of individual arguments...

Reply:
Where should the default location be then? In the project's root folder or some dedicated place?

Should I return the `githubConfig` struct that stored all these variables back then? Probably should rename it to something like `scmConfig` too.

Reply:
Yes. `scmConfig` is nice. And the folder could be `internal/config/github/quickfeed.pem` or something like that?